docs/src/core-concepts.mdx

```diff
@@ -11,6 +11,8 @@ import mobxTriad from './images/mobx-triad.png';
 
 At the heart of MobX are three important concepts: **Observables**, **Actions** and **Reactions**.
 
+> See the complete code [here](https://github.com/mobxjs/mobx.dart/tree/master/mobx_examples/lib/counter).
```

Comment:
Add this just before the Computed Observables section. Mention that this is part of the Counter example and give link to the example page on mobx.pub